### PR TITLE
bug: LT-1952 Update Third Party design for sub type & role

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: parties/serializers.py [hunk 1]
```diff
@@ -145,21 +131,11 @@ class Meta:
 
 
 class ThirdPartySerializer(PartySerializer):
-    sub_type = KeyValueChoiceField(choices=ThirdPartySubType.choices)
+    role = KeyValueChoiceField(choices=ThirdPartyRole.choices)
```
Comment (jfaracik) [inline parties/serializers.py]: Could you write custom errors for if the data isn't set

Comment (BenAndrew-Kainos) [reply parties/serializers.py]: For sure.

File: parties/migrations/0004_auto_20200113_1129.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.8 on 2020-01-13 11:29
+
+from django.db import migrations, models
```
Comment (stephanieboyd) [inline parties/migrations/0004_auto_20200113_1129.py]: Combine migrations

Comment (BenAndrew-Kainos) [reply parties/migrations/0004_auto_20200113_1129.py]: Done

**Pull request merged**